### PR TITLE
Add files via upload

### LINKED ISSUE #21: User-Dashboard: Wishlist(View product on wishlist)
- Develop the User Dashboard wishlist pages UI appropriately
- The user dashboard upload product page contains 3 pages ( you should send a pull request for each of the pages)
- Ensure they are responsive and follow the responsive design on the Figma
- Implement images properly
- Follow the colour guide and the appropriate colours

### PR DESCRIPTION
**Pull request by mnamesujit**
Created Password Reset UI Page Using HTML and CSS

## 🛠️ Fixes Issue

`Closes #21`
Closes #21

## 👨‍💻 Changes proposed

- Developed user dashboard: wishlist(view product on wishlist).
- All pages.
- Mobile responsive

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

Endpoints
- /wishlist (view products in wishlist)
- /add-to-wishlist (add products to wishlist)

## 📷 Screenshots

<br/>

<br/>

### CODE REVIEW

**Comment by mnamesujit** [comment]
I Have Created Reset Password UI Page Please Accept My pull request.

**Comment by ingressive4good** [comment]
@t-bello7 @dprincecoder

**Comment by dprincecoder** [comment]
This file can’t be merge right now, this is not a react functional component neither does it respect jsx syntax

File: Reset_password/index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                        <td style="height:80px;">&nbsp;</td>
+                    </tr>
+                </table>
+            </td>
+        </tr>
+    </table>
+    <!--/100% body table-->
+</body>
+
+</html>
```
[the remark this answers is not in the document]
Comment (dprincecoder) [reply Reset_password/index.html:81]: Yes no pure html to the project

**Comment by mnamesujit** [comment]
I am Created User Profile Card Using HTML and CSS

**Comment by mnamesujit** [comment]
[t-bello7](https://github.com/t-bello7) I didn't know React Sir So what should I do?

**Comment by ingressive4good** [comment]
@dprincecoder @t-bello7 what should we do to this?

**Comment by dprincecoder** [comment]
Okay no pacnic since you already created the html files and css
Just look at other peoples files how they set up their own files 
Use it then copy your own html files into the jsx file

**Comment by t-bello7** [comment]
We were all once at the HTML and CSS stage but using reacts comes with the advantage of handling states of the application gracefully. @mnamesujit Suggest you take it has an opportunity to learn react and tailwind. Everything you have written in the HTML file should be placed in the render function in your component. And the utility classes you created in the CSS file has already been created with tailwind integrated in the project. Learn how to implement tailwind utilities classes.

**Comment by dprincecoder** [comment]
@mnamesujit

**Comment by ingressive4good** [comment]
@dprincecoder @t-bello7 can you please refactor this to a JSX code

Thank you

**Comment by t-bello7** [comment]
The proposed issue of this PR is solving is already done

**Pull request closed without merging**